### DISCUSSION TITLE
What's VERGEN_GIT_SEMVER? How to upgrade?

### DISCUSSION THREAD

**Discussion by doki23** [databendlabs/databend#5812] [Q&A] [answered]
I cannot run query deployed in my local host because of not meeting the MIN_METASRV_SEMVER.
My VERGEN_GIT_SEMVER is 0.7.13-nightly and MIN_METASRV_SEMVER is 0.7.63
Please help.

**Comment by Xuanwo** [comment, marked as answer]
Please fetch all tags locally so that we got the correct git version:

```shell
git fetch --tags
```

For more information: https://databend.rs/doc/manage/upgrade/compatibility

**Reply by doki23** [reply]
Thx, it works. 
It's because my fork branch isn't sync with the upstream branch. 
The variant cmd is `git fetch <upstream> --tags`.

**Reply by Xuanwo** [reply]
Yep, it's always recommended to check out new branches from upstream instead.
